wolox-books/angular.json

```diff
@@ -72,7 +72,7 @@
             "tsConfig": "src/tsconfig.spec.json",
             "karmaConfig": "src/karma.conf.js",
             "styles": [
-              "src/styles.css"
+              "src/styles.scss"
```

Comment:
rename this for `src/scss/application.scss`, then rename the file and move it to a new folder all `scss` into `src`. In future prs you will have to create different files for more generic components